src/containers/IdentityContainer.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+
+import { esriPromise } from 'react-arcgis';
+
+import { Identity } from '../components/Identity';
+
+interface IdentityComponentProps {
+  credential?: __esri.Credential;
+  portalUrl: string;
+  handleLoginChange: (credential?: __esri.Credential) => void;
+}
+interface IdentityComponentState {}
+
+export class IdentityContainer extends React.Component<IdentityComponentProps, IdentityComponentState> {
+  identityManager: __esri.IdentityManager;
+
+  constructor(props: IdentityComponentProps) {
+    super(props);
+    this.handleLogin = this.handleLogin.bind(this);
+    this.handleLogout = this.handleLogout.bind(this);
+  }
+
+  componentDidMount() {
+    esriPromise([
+      'esri/identity/OAuthInfo',
+      'esri/identity/IdentityManager'
+    ]).then(([OAuthInfo, IdentityManager]) => {
+      this.identityManager = IdentityManager;
```

Comment:
```
<nitpick>This is a class (as opposed to an instance), so it should be capitalized</nitpick>
```

Reply:
Although, all the Esri examples don't capitalize it and actually rename the object to `esriId`:
```js
require(["esri/identity/OAuthInfo", "esri/identity/IdentityManager"], function(OAuthInfo, esriId) {
  var oAuthInfo = new OAuthInfo({
    appId: "<registered client id>"
  }); // required parameter
  esriId.registerOAuthInfos([oAuthInfo]);
});
```

Should we follow suit?

Reply:
This discussion reminds me a wee bit of single- vs. double quotes. I uppercased to `IdentityManager`. I didn't use `esriId` because I think it's not very descriptive.